speccurses.py

```diff
@@ -9,19 +9,8 @@
 import objects
 from objects import spec_decoder, spec_encoder, spec_config, spec_run
 
-TAB = 9
-ENTER = 10
-ESC = 27
-DOWN = 258
-UP = 259
-LEFT = 260
-RIGHT = 261
-HOME = 262
-BACK = 263
-DEL = 330
-PGDN = 338
-PGUP = 339
-END = 360
+#KEY_ENTER = 343, but my ENTER key registers as 10 =/
+KEY_ENTER = 10
```

Comment:
Are we looking at enter-key issues with different keyboards? Is there a different code entered for different systems?

Reply:
10 is ascii for \n. Curses doesn’t seem to pick up the actual enter key